rsocket-core/src/main/java/io/rsocket/core/RSocketRequester.java

```diff
@@ -208,7 +211,7 @@ public boolean isDisposed() {
       return Mono.error(t);
     }
 
-    if (!PayloadValidationUtils.isValid(this.mtu, payload)) {
+    if (!PayloadValidationUtils.isValid(this.mtu, payload, maxFrameLength)) {
```

Comment:
Shouldn't this be called `maxPayloadLength`? That's what's passed in. Same in `RSocketResponder`.

Reply:
That is different from `maxPayloadSize` fragmentation is aimed to maximum frame length that can be sent over the transport. If transport limits frame size to 128 bytes, then validation should be aligned with that. In contrast, `maxPayloadSize` is more general, since from 10 frames we can gather a single payload. `MaxPayloadSize` should always be >= to `maxFrameLength`

Reply:
I could make a mistake passing in payload size instead of maxFrameLenght configured in the transport

Reply:
Ok